src/main/java/seedu/address/model/ItemIndexWrapper.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model;
+
+import seedu.address.commons.core.item.Item;
+
+/**
+ * Stores an item along with its indices in the separate item lists
+ * */
+
+public class ItemIndexWrapper {
+    private Item item;
+    private int visual;
+    private int storage;
+    private int task;
+    private int eve;
+    private int rem;
+    private int frem;
```

Comment:
Do you want to elaborate on what these fields are and their purpose?

Reply:
alright, will do!